pytest-ginkgo.ini

```diff
@@ -0,0 +1,10 @@
+# This file supports PyTest testing Figures against Ginkgo mocks
+
+[pytest]
```

Comment:
`tox` should be used for tests multiplexing, I think it should be used instead of doing custom work like this.

Reply:
@OmarIthawi I looked at using env vars for `pytest.ini` The only project I found is no longer maintained. Do you know of another way besides env vars to set the `mocks/<platform>` directory to be a variable based on the tox environment for pytest?. For Ginkgo, we need to set:

```
[pytest]
...
python_paths = devsite mocks/ginkgo
```

and for Hawthorn, we need to set:

```
[pytest]
...
python_paths = devsite mocks/hawthorn
```

Reply:
@johnbaldwin I'm not sure what the question is, could you please explain?

In `tox.ini` files it's possible to manipulate the python path:

```ini
[testenv]
commman = pytest

[testenv:py27-ginkgo]
setenv =
    PYTHONPATH = devsite mocks/ginkgo

[testenv:py27-hawthorn]
setenv =
    PYTHONPATH = devsite mocks/hawthorn
```

Is that what you're looking for?

Reply:
nvm, that was not needed. Thanks John.